pkg/minishift/openshift/registry.go

```diff
@@ -51,10 +52,17 @@ func isRegistryRouteEnabled(registryAddonEnabled bool) bool {
 	return true
 }
 
-func fetchRegistryRoute() (string, error) {
+func fetchRegistryRoute(openshiftVersion string) (string, error) {
 	namespace := "default"
 	route := "route/docker-registry"
 	cmdArgText := fmt.Sprintf("get -o jsonpath={.spec.host}:443 %s -n %s --config=%s", route, namespace, constants.KubeConfigPath)
+	valid, err := openshiftVersionCheck.IsGreaterOrEqualToBaseVersion(openshiftVersion, "v3.7.0")
```

Comment:
@praveenkumar Can you add a comment saying what changed in v3.7.0? Also should n't we use a variable instead of directly using hard coded string "v3.7.0"?

Reply:
I mean `constants.BackwardIncompatibleOcVersion`.